<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowChannels</class>
 <widget class="QDialog" name="WindowChannels">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>323</width>
    <height>212</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Channels Window</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>1. Channel of image</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_1">
         <property name="editable">
          <bool>false</bool>
         </property>
         <property name="currentText">
          <string>Nucleus</string>
         </property>
         <property name="currentIndex">
          <number>1</number>
         </property>
         <item>
          <property name="text">
           <string>Brightfield</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Nucleus</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Cytoplasm</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>YAP</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Actin</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>None</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>2. Channel of image</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_2">
         <property name="editable">
          <bool>false</bool>
         </property>
         <property name="currentText">
          <string>Cytoplasm</string>
         </property>
         <property name="currentIndex">
          <number>2</number>
         </property>
         <item>
          <property name="text">
           <string>Brightfield</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Nucleus</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Cytoplasm</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>YAP</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Actin</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>None</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>3. Channel of image</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_3">
         <property name="editable">
          <bool>false</bool>
         </property>
         <property name="currentText">
          <string>YAP</string>
         </property>
         <property name="currentIndex">
          <number>3</number>
         </property>
         <item>
          <property name="text">
           <string>Brightfield</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Nucleus</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Cytoplasm</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>YAP</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Actin</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>None</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>4. Channel of image</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_4">
         <property name="editable">
          <bool>false</bool>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <item>
          <property name="text">
           <string>Brightfield</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Nucleus</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Cytoplasm</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>YAP</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Actin</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>None</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>5. Channel of image</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_5">
         <property name="editable">
          <bool>false</bool>
         </property>
         <property name="currentText">
          <string>None</string>
         </property>
         <property name="currentIndex">
          <number>5</number>
         </property>
         <item>
          <property name="text">
           <string>Brightfield</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Nucleus</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Cytoplasm</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>YAP</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Actin</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>None</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>131</width>
           <height>31</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="okButton">
         <property name="text">
          <string>OK</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>WindowChannels</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>WindowChannels</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
